src/test/java/com/wikia/webdriver/common/core/api/ApiCall.java

```diff
@@ -0,0 +1,78 @@
+package com.wikia.webdriver.common.core.api;
+
+import com.wikia.webdriver.common.core.Helios;
+import com.wikia.webdriver.common.core.annotations.User;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+
+import org.apache.http.NameValuePair;
+import org.apache.http.client.ClientProtocolException;
+import org.apache.http.client.entity.UrlEncodedFormEntity;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.openqa.selenium.WebDriverException;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by wikia on 2015-08-27.
+ */
+public abstract class ApiCall {
+
+  private static String ERROR_MESSAGE = "Problem with API call";
+
+  protected static String URL_STRING = null;
+  protected static User user = null;
+  protected static List<NameValuePair> nvps = null;
+
+  public ApiCall() {
+    user = null;
+    nvps = new ArrayList<>();
+  }
+
+  public static void call() {
+    try {
+      CloseableHttpClient httpClient = HttpClients.createDefault();
+      HttpPost httpPost = getHtppPost();
+      //set header
+      if (user != null) {
+        httpPost.addHeader("Authorization", "Bearer " + Helios.getAccessToken(user));
+      }
+      //set query params
+      if (nvps != null) {
+        httpPost.setEntity(new UrlEncodedFormEntity(nvps, StandardCharsets.UTF_8));
+      }
+
+      httpClient.execute(httpPost);
+
+      PageObjectLogging
+          .log("CONTENT PUSH", "Content posted to: " + URL_STRING,
+               true);
+    } catch (ClientProtocolException e) {
+      PageObjectLogging.log("EXCEPTION", e.toString(), false);
+      throw new WebDriverException(ERROR_MESSAGE);
+    } catch (IOException e) {
+      PageObjectLogging.log("IO EXCEPTION", e.toString(), false);
+      throw new WebDriverException(ERROR_MESSAGE);
+    } catch (URISyntaxException e) {
+      PageObjectLogging.log("URI_SYNTAX EXCEPTION", e.toString(), false);
+      throw new WebDriverException(ERROR_MESSAGE);
+    }
+  }
+
+  public static HttpPost getHtppPost() throws URISyntaxException, MalformedURLException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: java.net.URISyntaxException, java.net.MalformedURLException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1160)